server/controllers/UserController.js

```diff
@@ -153,7 +158,7 @@ class UserController {
             message: 'Incorrect Email or Password'
           });
         }
-      }).catch(err => err.message);
+      }).catch(err => console.log(err));
```

Comment:
#### Critical
Why do we have so many logs in this file? Remove all logging

Reply:
OK I will do that